server/sonar-server/src/main/java/org/sonar/server/es/BulkIndexer.java

```diff
@@ -70,32 +78,64 @@
   private final BulkProcessor bulkProcessor;
   private final AtomicLong counter = new AtomicLong(0L);
   private final SizeHandler sizeHandler;
+  private final BulkProcessorListener bulkProcessorListener;
+  @Nullable
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') org.sonar.server.es.BulkIndexer must not use org.elasticsearch.common.Nullable [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AElasticSearch_embeds_some_external_dependencies_which_should_never_be_used__Guava_and_Guice_for_instance__)